apps/climatemappedafrica/src/components/HURUmap/Panel/DesktopPanel/DesktopPanel.js

```diff
@@ -47,7 +47,7 @@ function DesktopPanel({ sx, ...props }) {
             overscrollBehaviorBlock: "none",
           },
         }}
-        variant="persistent"
+        {...(open && { variant: "persistent" })}
```

Comment:
I remember asking about the [this](https://github.com/CodeForAfrica/ui/pull/1039#discussion_r1940563033) change?

Reply:
@kilemensi 
1. We only have to update `handleClickMap` in Pesayetu. The reason is that Pesayetu uses the legacy `commons-ui`, hence no need to update the `LinkItem`  
2. It looks like the reason searching and clicking on the map is not working is a data issue: looking into it.

Reply:
And isn't link [`Link`](https://github.com/CodeForAfrica/ui/blob/main/apps/pesayetu/src/components/Link/index.js) included @kelvinkipruto ?

Reply:
> I remember asking about the [this](https://github.com/CodeForAfrica/ui/pull/1039#discussion_r1940563033) change?

@kilemensi Yes. This approach allows the search to work when the drawer is open, but we only need it to be persistent if the drawer is open. If we set it to persistent all the time, it will overlay over some elements, like the location tags, making them unclickable.

Reply:
> And isn't link [`Link`](https://github.com/CodeForAfrica/ui/blob/main/apps/pesayetu/src/components/Link/index.js?rgh-link-date=2025-02-24T13%3A26%3A08Z) included @kelvinkipruto ?

@kilemensi Yes. Pesayetu's `Link` uses the legacy [`commons-ui`](https://github.com/CodeForAfrica/ui/blob/1e0748c05b3d357ca95481349d984fc8274ddc2e/apps/pesayetu/src/components/Link/index.js#L1) which has no issue. ClimateMapped uses the new [`Link`](https://github.com/CodeForAfrica/ui/blob/1e0748c05b3d357ca95481349d984fc8274ddc2e/apps/climatemappedafrica/src/components/Link/index.js#L1) from `commons-ui/next` which always requires a `href` to be passed to it. In our case, we're not passing an `href` which causes it to break. Since we're using the `onClick` event we instead use a button.

Reply:
What am asking/saying @kelvinkipruto is:

Is it **us** or `mui` that requires `href` on `Link`? If it's us, that should be an easy fix to require either `href` or `onClick`. There is no need to say it's legacy `commons-ui`, etc. If it's not us, which is what I'm suspecting, then yes, lets document that `@mui/material/Link` _now_ requires `href`.

---

Even if it is indeed `mui`, can't we pass _fragment_ as `href` e.g. `/currentURL#new-code` since fragments don't cause server-side navigation? Then `onClick` should work fine.